examples/frontend_language/usage/srt_engine.py

```diff
@@ -0,0 +1,20 @@
+import sglang as sgl
```

Comment:
this is unrelated to the frontend language. Can we move this under `examples/runtime`?

Reply:
moved